src/main/python/community-detector/test_community_detector.py

```diff
@@ -0,0 +1,41 @@
+import unittest
+import os
+
+import numpy
+from numpy.testing import assert_array_equal
+from scipy.sparse import csr_matrix
+
+from community_detector import detect_communities
+
+dirname = os.path.dirname(__file__)
+
+
+def build_csr_matrix(input):
+    return csr_matrix(
+        (input['id_to_buckets_data'], input['id_to_buckets_indices'],
```

Comment:
I try to avoid using `input` as a variable name in python. Because it's built-in function: https://docs.python.org/3/library/functions.html#input

Reply:
Super useful heads up, thank you!